grunt_tasks/build.js

```diff
@@ -73,9 +73,22 @@ module.exports = function (grunt) {
                 files: [{
                     expand: true,
                     cwd: 'node_modules/jsoneditor/dist',
-                    src: ['img/**', 'jsoneditor.min.css'],
+                    src: ['img/**'],
                     dest: 'clients/web/static/built/jsoneditor'
                 }]
+            },
+            bootstrap: {
+                files: [{
+                    expand: true,
+                    cwd: 'node_modules/bootstrap/dist/css',
+                    src: ['bootstrap.min.css'],
+                    dest: 'clients/web/static/built'
```

Comment:
The path to bootstrap.min.css should be updated in:
- https://github.com/girder/girder/blob/6fe7e9538227939ce62f70e8bd83f08fb7d1af0e/clients/web-external/index.html#L2
- https://github.com/girder/girder/blob/6fe7e9538227939ce62f70e8bd83f08fb7d1af0e/docs/external-web-clients.rst#using-girder-register-and-login-ui